Adding the maintainers approved in 2023Q1 Core Maintainers meeting

gottbrath opened the pull request:
Added Nikita to Core Maintainers
Merged MKLDNN with CPU Performance
Renamed CUDA to GPU Performance
Added Jiong to Compiler and CPU Performance
Added Xiaobing to CPU Performance
Marking Vitaly and Jian Hui as Emeritus

gottbrath committed:
Adding the maintainers approved in 2023Q1 Core Maintainers meeting

Added Nikita to Core Maintainers
Merged MKLDNN with CPU Performance
Renamed CUDA to GPU Performance
Added Jiong to Compiler and CPU Performance
Added Xiaobing to CPU Performance
Marking Vitaly and Jian Hui as Emeritus

gottbrath committed:
Making the CPU and GPU performance modules parallel

Updating the name of the NVIDIA/CUDA module to be GPU performance to be parallel with CPU performance.

gottbrath committed:
Update persons_of_interest.rst

Missed an emeritus on Vitaly

dzhulgakov reviewed (commented):
Thanks! Should we update https://github.com/pytorch/pytorch/blob/master/.github/merge_rules.yaml in the same go?

gottbrath commented on docs/source/community/persons_of_interest.rst:
```diff
@@ -138,17 +140,22 @@ Fast Fourier Transform (torch.fft)
+-  (emeritus) Jianhui Li (`Jianhui-Li <https://github.com/Jianhui-Li>`__)
 
-NVIDIA / CUDA
+GPU Performance (Torch Inductor / Triton / NVIDIA / CUDA )
```
In that same line of thinking should put the ROCm maintainers in this group as well?

gottbrath commented:
> Thanks! Should we update https://github.com/pytorch/pytorch/blob/master/.github/merge_rules.yaml in the same go?

Good catch. Yes probably. I am not as familiar with that, but it looks fairly straightforward. Let me give it a stab.

gottbrath commented:
Just tagging that Mike Brown noticed that the Readme.md file had some out of date info in https://github.com/pytorch/pytorch/issues/98456

gottbrath committed:
Update README.md

updated the maintainers list to match the core maintainers. Thanks to Mike Brown for noticing that it was out of date (see https://github.com/pytorch/pytorch/issues/98456)

gottbrath committed:
Update persons_of_interest.rst

dropping NVIDIA from this list per suggestion from dzhulgakov

gottbrath committed:
Update merge_rules.yaml

updated the core maintainers section to add Nikita's name to the list.

gottbrath commented:
made some updates. 

there is still some rationalization that we need to do around

Module structure (we should probably merge the rocm in with the Compiler GPU backend ) 
module level structure should be aligned with the yaml file
The thanks list in the readme.md seems quite out of date.

gottbrath committed:
Update persons_of_interest.rst

fixing a warning that came out in testing related to length of the title underlines.

gottbrath committed:
Update persons_of_interest.rst

removing trailing spaces that were failing a lint test

dzhulgakov commented on README.md:
```diff
@@ -435,7 +435,7 @@ To learn more about making a contribution to Pytorch, please see our [Contributi
 
-PyTorch is currently maintained by [Adam Paszke](https://apaszke.github.io/), [Sam Gross](https://github.com/colesbury), [Soumith Chintala](http://soumith.ch) and [Gregory Chanan](https://github.com/gchanan) with major contributions coming from hundreds of talented individuals in various forms and means.
+PyTorch is currently maintained by [Soumith Chintala](http://soumith.ch), [Gregory Chanan](https://github.com/gchanan), [Dmytro Dzhulgakov](https://github.com/dzhulgakov), [Edward Yang](https://github.com/ezyang), and [Nikita Shulga](https://github.com/malfet) with major contributions coming from hundreds of talented individuals in various forms and means.
 A non-exhaustive but growing list needs to mention: Trevor Killeen, Sasank Chilamkurthy, Sergey Zagoruyko, Adam Lerer, Francisco Massa, Alykhan Tejani, Luca Antiga, Alban Desmaison, Andreas Koepf, James Bradbury, Zeming Lin, Yuandong Tian, Guillaume Lample, Marat Dukhan, Natalia Gimelshein, Christian Sarofeen, Martin Raison, Edward Yang, Zachary Devito.
```
Hm, it's indeed quite out of date. Maybe link to PT paper (https://arxiv.org/pdf/1912.01703.pdf) it has the original list of acknowledgments and persons_of_interest.rst for module-level

gottbrath replied:
I do think we should do that in a follow up PR.

dzhulgakov commented on docs/source/community/persons_of_interest.rst:
```diff
@@ -138,18 +140,23 @@ Fast Fourier Transform (torch.fft)
 
--  Vitaly Fedyunin (`VitalyFedyunin <https://github.com/VitalyFedyunin>`__)
 -  Mingfei Ma (`mingfeima <https://github.com/mingfeima>`__)
+-  Jiong Gong (`jgong5 <https://github.com/jgong5>`__)
+-  Xiaobing Zhang (`XiaobingSuper <https://github.com/XiaobingSuper>`__)
```
looks like these folks were already in merge_rules.yaml's sections of "CPU ATen backend" and "CPU frontend", so they have merge access already :)

gottbrath commented:
@pytorchbot merge

gottbrath commented:
ok, I don't think this needs an additional release note. So while this is user facing I'll go ahead and add the not user facing tag.

gottbrath commented:
@pytorchbot merge

gottbrath commented:
@pytorchbot merge -r

gottbrath commented:
@pytorchbot merge
